packages/common-all/src/dnode.ts

```diff
@@ -1185,16 +1185,24 @@ export class NoteUtils {
    * @param fname The fname of note that you want to get the color of.
    * @returns The color, and whether this color was randomly generated or explicitly defined.
    */
-  static color(opts: { fname: string; vault?: DVault }): {
+  static color(opts: {
+    fname: string;
+    vault?: DVault;
+    note?: NoteProps | NotePropsMeta;
```

Comment:
nit: `note?: NotePropsMeta` should be sufficient here since the type is a strict subset of `NoteProps`